tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -566,6 +566,87 @@ describe('migrate-converged-pkg generator', () => {
     });
   });
 
+  describe(`npm config setup`, () => {
+    it(`should update .npmignore config`, async () => {
+      function getNpmConfig(projectConfig: ReturnType<typeof readProjectConfiguration>) {
+        return tree.read(`${projectConfig.root}/.npmignore`)?.toString('utf-8');
+      }
+      const projectConfig = readProjectConfiguration(tree, options.name);
+      let npmConfig = getNpmConfig(projectConfig);
+
+      expect(npmConfig).toMatchInlineSnapshot(`
+        "*.api.json
+        *.config.js
+        *.log
+        *.nuspec
+        *.test.*
+        *.yml
+        .editorconfig
+        .eslintrc*
+        .eslintcache
+        .gitattributes
+        .gitignore
+        .vscode
+        coverage
+        dist/storybook
+        dist/*.stats.html
+        dist/*.stats.json
+        dist/demo
+        fabric-test*
+        gulpfile.js
+        images
+        index.html
+        jsconfig.json
+        node_modules
+        results
+        src/**/*
+        !src/**/examples/*.tsx
+        !src/**/docs/**/*.md
+        !src/**/*.types.ts
+        temp
+        tsconfig.json
+        tsd.json
+        tslint.json
+        typings
+        visualtests"
+      `);
+
+      await generator(tree, options);
+
+      npmConfig = getNpmConfig(projectConfig);
+
+      expect(npmConfig).toMatchInlineSnapshot(`
+        ".cache/
+        .storybook/
+        .vscode/
+        coverage/
+        src/
+        bundle-size/
+        config/
+        temp/
+        e2e/
+        node_modules/
+        __fixtures__
+        __tests__
+
+        *.log
+        *.yml
+        *.test.*
+        *.spec.*
+        *.stories.*
+        *.api.json
```

Comment:
[`CHANGELOG.*` files](https://cdn.jsdelivr.net/npm/@fluentui/babel-make-styles@9.0.0-alpha.33/CHANGELOG.md) will be still shipped, right? Is it expected?

Reply:
ofc, that's one of few defaults that should be always shipped (Changelog,Licencse,Readme)

- also if `files` are used those afformentioned files are alwyas shipped by npm no matter what  (files was not the right way to go with - described in RFC )